modules/platforms/cpp/odbc-test/CMakeLists.txt

```diff
@@ -62,7 +62,7 @@ set(SOURCES src/teamcity/teamcity_boost.cpp
         src/errors_test.cpp
         src/odbc_test_suite.cpp
         src/types_test.cpp
-        src/transaction_test.cpp
+#        src/transaction_test.cpp
```

Comment:
Whats wrong with CPP tests?
Is there any ticket for fixing them?

Reply:
transaction_test.cpp uses TRANSACTIONAL_SNAPSHOT mode. As MVCC is deprecated I want to ignore this tests, but I've failed to find the right way to ignore them. So I comment them for a while. There are a lot of java tests that marked with Ignore now, I think can we just remove those tests at all? If yes I'd like to completely remove this CPP test too.